.env.example

```
@@ -33,9 +33,9 @@ L2PS_HASH_INTERVAL_MS=5000
 # ===========================================
 # OmniProtocol TCP Server Configuration
 # ===========================================
-OMNI_ENABLED=false
+OMNI_ENABLED=true
 OMNI_PORT=3001
-OMNI_MODE=OMNI_ONLY
+OMNI_MODE=OMNI_PREFERRED
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OMNI_MODE key should go before the OMNI_PORT key

(UnorderedKey)